README.md

```diff
@@ -26,10 +26,9 @@ to DataLad. In order to install DataLad customized for a particular domain, one
 can simply install an extension directly, and DataLad itself will be
 automatically installed with it. Here is a list of known extensions:
 
-- [tracking web resources and automated data distributions](https://github.com/datalad/datalad-crawler) [![crawler release](https://img.shields.io/github/release/datalad/datalad-crawler.svg)](https://GitHub.com/datalad/datalad-crawler/releases/)
-- [neuroimaging research data and workflows](https://github.com/datalad/datalad-neuroimaging) [![neuroimaging release](https://img.shields.io/github/release/datalad/datalad-neuroimaging.svg)](https://GitHub.com/datalad/datalad-neuroimaging/releases/)
-- [support for containerized computational environments](https://github.com/datalad/datalad-container) [![container release](https://img.shields.io/github/release/datalad/datalad-container.svg)](https://GitHub.com/datalad/datalad-container/releases/)
-- [alternative set of basic commands with improved cross-platform support](https://github.com/datalad/datalad-revolution) [![revolution release](https://img.shields.io/github/release/datalad/datalad-revolution.svg)](https://GitHub.com/datalad/datalad-revolution/releases/)
+- [crawler -- tracking web resources and automated data distributions](https://github.com/datalad/datalad-crawler) [![crawler release](https://img.shields.io/github/release/datalad/datalad-crawler.svg)](https://GitHub.com/datalad/datalad-crawler/releases/)
```

Comment:
minor style nit: I think this would look better if the link description included only the name (e.g.
`[crawler](https://github.com/datalad/datalad-crawler)  -- tracking web resources and automated data distributions`).

Reply:
Yep, will do

Reply:
done